pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -54,14 +54,14 @@ let
 
     strictDeps = true;
 
+    __structuredAttrs = true;
```

Comment:
why this change ?

Reply:
To avoid special cases for structured attributes by having them always enabled. Now that I think about it, probably makes more sense to use `configureScript` as a wrapper instead of overriding `configurePhase`.

Reply:
Updated the PR to use `configureScript`.